src/model_repository_manager/model_repository_manager.cc

```diff
@@ -1401,17 +1437,22 @@ ModelRepositoryManager::InitializeModelInfo(
     // the override while the local files may still be unchanged.
     linfo->mtime_nsec_ = std::make_pair(0, 0);
     linfo->model_path_ = location;
+    linfo->model_config_path_ = JoinPath({location, kModelConfigPbTxt});
```

Comment:
Should be like below as well..
```
linfo->model_config_path_ =
        GetModelConfigFullPath(linfo->model_path_, model_config_name_);`
```

Reply:
Can you add a test case where `configs/<some_config_name>.pbtxt` is specified via loading with file override?

Reply:
Correct me if I am wrong. In this case, `LocalizeRepoAgent` in line 1417 will create the model config file named `kModelConfigPbTxt` aka "config.pbtxt" in a tmp directory. Therefore, the config file path is static.

Reply:
Looks safe to me. See error below when not specifying config for model file override.
`E0516 17:57:40.808915 923786 model_repository_manager.cc:1371] Poll failed for model directory 'onnx_float32_float32_float32': Load parameter 'config' must be specified for model file override`